common/static/common/js/spec/components/paging_collection.js

```diff
@@ -32,6 +32,44 @@ define(["backbone.paginator", "backbone"], function(BackbonePaginator, Backbone)
             this.currentPage = currentPage;
             this.start = start;
             return response.items;
+        },
+
+        setPage: function (page) {
```

Comment:
In fact, it looks like it is no longer used. Before your changes, it was only referenced by paging_spec.js, but that now uses an actual paging collection. Can this be removed now?